### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by alexdor**

### CODE REVIEW

File: src/data/communication-channels/communication-channels.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 [
   {
     "title": "Slack",
-    "link": "#",
+    "link": "https://mossn.slack.com",
     "attachment": "./icon-slack.svg"
   },{
     "title": "Email",
-    "link": "#",
+    "link": "ossn@mozilla.com",
```
Comment (alexdor) [inline src/data/communication-channels/communication-channels.json]: This isn't working, it redirects you to a 404 page

File: src/components/components/promoted-box/promoted-box.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default props => {
           <li className="promoted-box__item">
             Explore contribution opportunities
           </li>
-          <li className="promoted-box__item"> Reach out to us </li>
+          <li className="promoted-box__item"> Join our Slack instance </li>
```
Comment (alexdor) [inline src/components/components/promoted-box/promoted-box.js]: Shouldn't this be a link?

File: src/components/components/footer/footer.js [hunk 3]
```diff
@@ -132,7 +132,7 @@ const footerNavigationList1 = [
   },
   {
     title: 'Affiliations',
-    target: '/affiliations'
+    target: '/organizations'
```
Comment (alexdor) [inline src/components/components/footer/footer.js]: This is confusing I think, shouldn't the `/affiliations` change to point where the `/organizations` is pointing now?

File: src/data/image-box/image-box.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
   "box0": [
     {
-      "titleLargeTop": "Read",
-      "titleSmall": "our updated",
+      "titleLargeTop": "March 1st",
+      "titleSmall": "launching our",
       "titleLargeBottom": "blog!",
       "link": "#",
```
Comment (alexdor) [inline src/data/image-box/image-box.json]: Shouldn't this point somewhere?

File: src/components/components/promoted-box/promoted-box.js (hunk 2 above)
Comment (bacharakis) [reply src/components/components/promoted-box/promoted-box.js]: Nope, that's fine

File: src/components/components/footer/footer.js (hunk 3 above)
Comment (bacharakis) [reply src/components/components/footer/footer.js]: Yes, but it's P2

**Pull request merged**